src/components/VideoCard/index.jsx

```diff
@@ -1,7 +1,5 @@
 import React from "react";
 import styles from "./style.module.css";
```

Comment:
video card missing a cursor value of pointer, not sure if that was intentionally kept like that

Reply:
will update this